src/Props/teamCardProps/cards.ts

```diff
@@ -27,14 +27,20 @@ const cards: CardProps[] = [
     LinkedInLink: "https://linkedin.com/in/hannah-wong-76a604173/",
   },
   {
-    name: "Tracy Rocha",
+    name: "Navy Rao",
     title: "Director of Design",
     imgSrc: cat,
+    LinkedInLink: "https://www.linkedin.com/in/navya-rao-682140142/",
+  },
+  {
+    name: "Tracy Rocha",
+    title: "Director of Logistics",
+    imgSrc: cat,
     LinkedInLink: "https://www.linkedin.com/in/tracy-a-rocha/",
   },
   {
     name: "Katie Allen",
-    title: "Director of Logistics",
+    title: "Director of Sponsorship",
     imgSrc: cat,
     LinkedInLink: "https://www.linkedin.com/in/kaitlynallen/",
   },
```

Comment:
Almost everyone has posted their profile picture into google drive so let's start adding those as well so we can crop or change those images if needed